go/worker/common/committee/group.go

```diff
@@ -255,6 +255,11 @@ type Group struct {
 	logger *logging.Logger
 }
 
+// RoundContext returns the active epoch's round context.
+func (g *Group) RoundContext() context.Context {
+	return g.activeEpoch.roundCtx
```

Comment:
Access to `g.activeEpoch` requires the read lock to be held. Also what happens in case `g.activeEpoch` is `nil`?

Reply:
Oops, will add the lock.  Where this is called from, the `activeEpoch` shouldn't be `nil`, though.